charts/trusted-artifact-signer/values.yaml

```diff
@@ -5,6 +5,20 @@ global:
   appsSubdomain: ""
 
 configs:
+  tas_initializer_job:
+    enabled: true
+    namespace: "rekor-system"
+    values:
+      organization_name: "trusted-artifact-signer"
+      email_address: "trustedartifactsigner@redhat.com"
+      password: "super-secret-password"
+    image:
+      registry: quay.io
+      repository: grpereir/tas-initializer-job
```

Comment:
Presumably this will change ;)

Reply:
This will be changed, however after the PR gets merged. When the new image builds. Those secret values dont matter if they get exposed, these are just defaults.